### DISCUSSION TITLE
I seem to be able to push commits to a locked branch

### DISCUSSION THREAD

**Discussion by PieterjanDeClippel** [community/community#49766] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I didn't change anything in my repository settings for a while, and all of a sudden I seem to be able to push (not force) commits to the `master` branch.

The `master` branch has Branch Protection Rules setup with the following settings:
- Require a pull request before merging
    - Require approvals
- Lock branch
- [Not] allow force pushes

However I'm still able to push commits to the master branch. Note that my laptop env git user is the same as the administrator of the organization owning the repository.

**Comment by bbkx226** [comment, marked as answer]
It's possible that you have been granted an exemption to the branch protection rules for the repository, or that the rules have been temporarily disabled. You should check with the repository owner or administrator to confirm if this is the case.

Alternatively, it's possible that the branch protection rules have been set up incorrectly or have been changed without your knowledge. You can check the repository's settings to ensure that the rules are still in place and configured correctly.

To check the branch protection rules for the repository, follow these steps:

1. Navigate to the repository on GitHub.
2. Click on the "Settings" tab.
3. Click on "Branches" in the left-hand sidebar.
4. Check the settings for the "master" branch to ensure that the branch protection rules are still in place and configured correctly.

If you find that the rules have been changed or disabled, you should reach out to the repository owner or administrator to discuss why this has occurred and how to address it.

**Reply by PieterjanDeClippel** [reply]
Thanks

**Reply by bbkx226** [reply]
A pleasure

**Comment by 03timothyash** [comment]
Chromecast

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 03timothyash** [comment]
Listo gracias por trabajar junto ami

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
